operating_unit/security/operating_unit_security.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+
+    <record model="ir.module.category" id="module_operating_units">
+        <field name="name">Operating Units Group</field>
+        <field name="sequence">30</field>
+    </record>
+
+    <record id="group_multi_operating_unit" model="res.groups">
+        <field name="name">Multiple Operating Unit</field>
+        <field name="category_id" ref="module_operating_units"/>
+    </record>
+
+    <record id="group_manager_operating_unit" model="res.groups">
```

Comment:
I like this new group added in this version. I think we can propose this to previous versions. That will solve #129

Reply:
It is easy said than done. Applying those changes to v11/v10 will man to make global rules to a simple group rules and cause many errors in database with the modules installed. I changed my mind, and better to keep this from v12 onwards.